src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala

```diff
@@ -81,6 +81,8 @@ object TpolecatPlugin extends AutoPlugin {
     ScalacOption("-Ywarn-unused-import", addedIn = Some(11), removedIn = Some(12)), // Warn if an import selector is not referenced.
     ScalacOption("-Ywarn-unused:implicits", addedIn = Some(12), removedIn = Some(13)),           // Warn if an implicit parameter is unused.
     ScalacOption("-Wunused:implicits", addedIn = Some(13), removedIn = dotty),                   // ^ Replaces the above
+    ScalacOption("-Ywarn-unused:explicits", addedIn = Some(13), removedIn = dotty),              // Warn if an explicit parameter is unused.
```

Comment:
I don't know if this old style "-Ywarn-unused" option is needed as I think that the `explicits` option was added after the new style of options was added?

Reply:
I've removed it, we can revert the specific commit if it's ever required

thanks